__tests__/HomePage.test.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import Homepage from "@/pages/Homepage";
```

Comment:
Please change this alias, this is not readable. You can create an alias for pages